scout/server/blueprints/variants/controllers.py

```diff
@@ -1071,6 +1071,10 @@ def check_form_gene_symbols(
     clinical_hgnc_ids = store.clinical_hgnc_ids(case_obj)
     clinical_symbols = store.clinical_symbols(case_obj)
 
+    # if no clinical symobols / panels were found loaded, warnings are treated as with research
+    if len(clinical_hgnc_ids == 0) and len(clinical_symbols == 0):
```

Comment:
You could have also checked if the case had gene panels, but this works as well! 👍🏻

Reply:
Yeah, and that could then be done before the other two checks. But then one would have to preset them to like empty lists aso.. CodeFactor is not screaming about the complexity yet. 😬